pkgs/servers/etcd/3.5/update.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq nix-prefetch
+
+set -x -eu -o pipefail
+
+ETCD_VERSION_MAJOR_MINOR=3.5
```

Comment:
Why not generalize this and get the version based on the file?

Reply:
That is the goal for next release. I have tested script for 3.5 only. I can't abstract it while I'm not sure of what will change for next release. For 3.4, I don't plan adding the update script, since it is going to be removed soon. So it's an incremental improvement.

Reply:
Great idea. Thanks.